Action/NotifyAction.php

```diff
@@ -44,7 +44,9 @@ public function execute($request)
             $request->setModel($refundPayU->getModel());
         } else {
             $setPayU = new SetPayU($request->getToken());
-            $setPayU->setModel($request->getModel());
+	    $model = $request->getModel();
```

Comment:
Formatting seems off.

Reply:
still looks a little bit off here but after opening full file it looks ok https://github.com/accesto/PayumPayU/blob/e13bc717c5af21408b980f88144c6fdc563806cf/Action/NotifyAction.php